modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/HeterogenousCRSTest.java

```diff
@@ -98,6 +94,7 @@ public static void initCRS() {
         System.setProperty("org.geotools.referencing.forceXY", "true");
     }
 
+    @Ignore
```

Comment:
@dvntucker - I have marked this test as ignored for now - you had mentioned it was rather brittle. Now that I have everything else working on Java 9, did you want to take a look at it when you get the chance?